src/node_constants.cc

```diff
@@ -817,6 +817,12 @@ void DefineCryptoConstants(Local<Object> target) {
     NODE_DEFINE_CONSTANT(target, OPENSSL_VERSION_NUMBER);
 #endif
 
+#ifdef OPENSSL_IS_BORINGSSL
+#define OPENSSL_IS_BORINGSSL true
```

Comment:
`NODE_DEFINE_CONSTANT()` expects a numeric value, not a boolean. You're also re-defining `OPENSSL_IS_BORINGSSL` here, which typically results in a compiler warning (that you can disable by `#undef`ing it first)